pkg/pkg.ml

```diff
@@ -33,7 +33,10 @@ let () =
     Ok [ Pkg.lib "pkg/META.mirage-types" ~dst:"META";
          Pkg.lib "types/V1.mli";
          Pkg.lib "types/V1.cmi";
+         Pkg.lib "types/V1.cmti";
          Pkg.lib ~cond:lwt "types/V1_LWT.mli";
-         Pkg.lib ~cond:lwt "types/V1_LWT.cmi"; ]
+         Pkg.lib ~cond:lwt "types/V1_LWT.cmi";
+         Pkg.lib ~cond:lwt "types/V1_LWT.cmti";
+         Pkg.mllib "types/mirage-types-pp.mllib"; ]
```

Comment:
but I couldn't figure out how to put V1 in the mllib without an implementation, and V1_LWT only conditionally...